src/nodes.coffee

```diff
@@ -3097,6 +3114,13 @@ exports.If = class If extends Base
 
 UTILITIES =
   modulo: -> 'function(a, b) { return (+a % (b = +b) + b) % b; }'
+  _boundMethodCheck: -> "
```

Comment:
@GeoffreyBooth I think @connec was mimicking babel's `_classCallCheck`. Updated to `boundMethodCheck`